### DISCUSSION TITLE
Can be `getStaticPaths` fallback rendered server side?

### DISCUSSION THREAD

**Discussion by neupauer** [vercel/next.js#17427] [Help] [answered]
Hey 👋, I'd like to ask if there is an option to render fallback static paths on the server (SSR)?

Imagine a lot of items (e.g., 100 000, not frequently changed) that can not be prerendered on build time, but you always want to get static HTML.

So, if the page (static path) is not yet generated (first visit), it would behave like `getServerSideProps` but when the page is generated it will be returned immediately (second visit).

Imagine a setup like this:
```js
export async function getStaticPaths() {
  return { paths: [], fallback: true };
}

export async function getStaticProps({ params }) {
  const item = await fetchItem(params.id);
  return { props: { item } };
}
```

so, my question is, is there an option something like this or a way to achieve this?

```js
export async function getStaticPaths() {
  return { paths: [], fallback: true, SSR: TRUE };
}
```

Thanks.

**Comment by lfades** [comment, marked as answer]
You're looking for [`fallback: 'blocking'`](https://github.com/vercel/next.js/pull/15672). Currently an experimental feature but you're welcome to try it out!

```jsx
export async function getStaticPaths() {
  return { paths: [], fallback: 'unstable_blocking' };
}
```

**Reply by neupauer** [reply]
That's exactly what I was looking for. Thank you 🙏.
